File: LeetCode/1295.cpp
//Link: https://leetcode.com/problems/find-numbers-with-even-number-of-digits/

class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int count = 0, digit;
        for(int n : nums) {
            digit = 0;
            while(n != 0) {
                n /= 10;
                digit += 1;
            }
            if((digit & 1) == 0)
                count += 1;
        }
        return count;
    }
};

File: LeetCode/1295.py
#Link: https://leetcode.com/problems/find-numbers-with-even-number-of-digits/

class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        count = 0
        for i in nums:
            string = str(i)
            if((len(string) & 1) == 0):
                count += 1
        return count
        